scripts/validate-created.py

```diff
@@ -0,0 +1,52 @@
+"""
+Validates the 'Created: ' field of PEPs match %d-%b-%Y as specified in PEP 1.
+
+Requires Python 3.9+
+
+Usage: python3 validate-created.py filename1 [filename2 [...]]
+"""
+import sys
+from datetime import datetime as dt
+
+COL_NUMBER = len("Created:")
+
+
+def validate_file(filename: str) -> int:
+    errors = 0
+    found = False
+    line_number = 0
+
+    with open(filename, encoding="utf8") as f:
+        for line in f:
+            line_number += 1
+            if line.startswith("Created:"):
+                found = True
+                break
+
+    if not found:
+        errors += 1
+        print(f"{filename}: 'Created:' not found")
+        return errors
+
+    date = line.removeprefix("Created:").strip()
+    try:
+        # Validate
+        dt.strptime(date, "%d-%b-%Y")
```

Comment:
The `"%d-%b-%Y"` format depends on the current locale; it might be different across machines.

Reply:
Ah right, your suggestion at https://github.com/python/peps/pull/1805#issuecomment-775041278 for something like `%Y-%m-%d` (`2021-02-08`) would help with this.